### PR TITLE
fix spelling mistakes, fix broken link

### PR DESCRIPTION
**Pull request by realsobek**
for siteindex.html: I hope the solution to fix the link is OK

### CODE REVIEW

File: download.md [hunk 1]
```diff
@@ -60,8 +60,8 @@ first time you run it.
 | [Linux Flatpak](https://flathub.org/apps/details/org.shotcut.Shotcut) | [64-bit Linux Snap](https://snapcraft.io/shotcut)
 {:.withborders}
 
-On [snap-enabled systems](https://snapcraft.io/docs/core/install), install the
-from the store with `snap install shotcut --classic`
```
Comment (ddennedy) [inline download.md]: Thank you for the help. The two trailing spaces at the end of this line are significant. They introduce a line break.

File: siteindex.html [hunk 2]
```diff
@@ -27,7 +27,7 @@ <h5>About</h5>
           <p><a href="{{ site.baseurl }}{{ page.url }}">{{ page.title }}</a></p>
         {% endif %}
       {% endfor %}
-      <a href="{{ site.baseurl }}blog/">News (Blog)</a><br>
+      <a href="{{ site.baseurl }}../blog/">News (Blog)</a><br>
```
Comment (ddennedy) [inline siteindex.html]: This should be "{{ site.baseurl }}/blog/"

**Review by ddennedy — CHANGES_REQUESTED** [review]

**Comment by realsobek** [comment]
thank you for feedback and explanation; the changes have been incorporated

**Review by ddennedy — APPROVED** [review]

**Comment by ddennedy** [comment]
Thank you. These changes are live now.

**Pull request merged**